dashboard/app/helpers/levels_helper.rb

```diff
@@ -91,7 +91,7 @@ def select_and_remember_callouts(always_show = false)
   # Options hash for all level types
   def app_options
     # Provide the channel for templated and applab levels.
-    set_channel if @level.try(:project_template_level) || @level.game == Game::APPLAB
+    set_channel if @level.try(:project_template_level) || @level.game == Game.applab
```

Comment:
project_template_level not project_level_template? The method is defined on Level https://github.com/code-dot-org/code-dot-org/blob/10f2e292d04c769f37b9f5edb148b7cb0a3f2aa3/dashboard/app/models/level.rb#L190 so I don't see why it would say undefined method (it could certainly return nil but that should be fine)